inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,412 @@
+
+prizes:
+    """Contains prize name.
+
+    In JSON stored as a list of strings.
+
+    TO DO: define a type inside list
+    """
+    schema:
+        {'prizes': {'type': 'list', 'force': True}}
+    creator:
+        marc, "678__", value['a']
+    producer:
+        json_for_marc(), {"678__": ""}
+
+author_note:
+    """Contains public note."""
+    derived:
+        @parse_first('_public_note', '_private_note', '_curators_note')
+        @only_if('_public_note' in self or '_private_note' in self
+                 or '_curators_note' in self)
+        sum_notes(self, '_public_note', '_private_note', '_curators_note')
+
+_public_note:
+    schema:
+        {'_public_note': {'type': 'string'}}
+    creator:
+        marc, "680__", value['i']
+    producer:
+        json_for_marc(), {"680__", ""}
+
+_private_note:
+    schema:
+        {'_private_note': {'type': 'string'}}
+    creator:
+        marc, "595__", value['a']
+    producer:
+        json_for_marc(), {"595__", ""}
+
+_curators_note:
+    schema:
+        {'_curators_note': {'type': 'string'}}
+    creator:
+        marc, "667__", value['a']
+    producer:
+        json_for_marc(), {"667__", ""}
+
+experiments:
+    """Contains information about experiments.
+
+    Currently the ``id`` subfield stores the name of the experiment. It should
+    be moved to storing id of the experiment as soon as experiments are
+    available as records.
+    """
+    schema:
+        {'experiments': {'type': 'list', 'force': True, 'items': {
+         'id': {'type': 'string'},
+         'start_year': {'type': 'integer', 'nullable': True,
+                        'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                        'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+         'end_year': {'type': 'integer', 'nullable': True,
+                      'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                      'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+         'status': {'type': 'string', 'nullable': True,
+                    'allowed': __import__('flask').current_app.config['INSPIRE_EXPERIMENT_STATUS']}
+         }}}
+    creator:
+        marc, "693__", {'id': value['e'], 'start_year': to_int(value['s']),
+                        'end_year': to_int(value['d']), 'status': value['z']}
+    producer:
+        json_for_marc(), {"693__e": "id", "693__s": "start_year",
+                          "693__d": "end_year", "693__z": "status"}
+
+
+phd_advisors:
+    """Contains a links to the advisors in form of record ids.
+
+    TO DO: define a type for the elements (int). Add creator and producer.
+    """
+    schema:
+        {'phd_advisors': {'type': 'list', 'force': True}}
+
+conferences:
+    """Contains information about conferentions attended.
+
+    TO DO: creator and producer. Move to records.
+    """
+    schema:
+        {'conferences': {'type': 'list', 'force': True}}
+
+urls:
```

Comment:
<datafield tag="856" ind1="4" ind2=" ">
<subfield code="u">https://twitter.com/BobbyAcharya</subfield>
<subfield code="y">TWITTER</subfield>
</datafield>

Add a subfield for the label/description of the link